### PR TITLE
feat: new metadata models endpoints

### PR DESCRIPTION
**Pull request by lukeocodes**
adds new metadata get model endpoints to the JS SDK

### CODE REVIEW

File: test/manage_rest.test.ts [hunk 1]
```diff
@@ -228,4 +228,23 @@ describe("making manage REST requests", () => {
     assert.isNotNull(result);
     assert.containsAllDeepKeys(result, ["balance_id"]);
   });
+
+  it("should retrieve a list of models", async () => {
+    const { result, error } = await deepgram.manage.getAllModels(faker.string.uuid());
+
+    assert.isNull(error);
+    assert.isNotNull(result);
+    // Add your assertions for the response data here
```
Comment (naomi-lgbt) [inline test/manage_rest.test.ts:237]: Do we want to test that a specific property is present, like we do in the other tests?

**Review by naomi-lgbt — APPROVED** [review]
This looks good! One non-blocking question.

File: test/manage_rest.test.ts (hunk 1 above)
Comment (lukeocodes) [reply test/manage_rest.test.ts:237]: shhhhh i hate this logic 

but yes we do :(

**Comment by jpvajda** [comment]
@naomi-lgbt  @lukeocodes  How close are we to getting this merged in?

**Comment by lukeocodes** [comment]
@naomi-lgbt or @dvonthenen if you could review/approve. I have updated the tests to actually check the response body as requested, and commits now squashed

cc @jpvajda

**Review by dvonthenen — APPROVED** [review]

**Pull request merged**